Products/Zuul/facades/mibfacade.py

```diff
@@ -74,6 +74,22 @@ def oidcmp(self, node1, node2):
         # This case occurs when one OID is the parent of another
         return cmp( len(a), len(b) )
 
+    def _searchDifferentOrganizers(self, uid, oid):
+        """
+        Check if OID exists in different MIB Organizer
+        """
+        organizer = self._dmd.getObjByPath(uid).id
+        oid_list = self._dmd.Mibs.mibSearch(oid=oid)
+        for oid in oid_list:
+            oid_obj = oid.getObject()
+            if oid_obj.moduleName != organizer:
+                oid_data = {'old_oid': oid_obj.oid,
+                            'old_id': oid_obj.id,
+                            'old_moduleName': oid_obj.moduleName,
+                            'new_moduleName': organizer }
+                return oid_data
```

Comment:
Facades in Zenoss, in general, should be returning `IInfo` objects, not custom dictionaries.